src/getexchange.c

```diff
@@ -615,277 +770,34 @@ void checkexchange(char *comment) {
 	}
     }
 
-    // -----------------------------------cqww-----------------------
-    if (CONTEST_IS(CQWW)) {
-
-	s = atoi(comment);
-	snprintf(zone, sizeof(zone), "%02d", s);
-
-	for (ii = 0; ii < LEN(zonepats); ii++) {
-
-	    hr = getlastpattern(zonepats[ii]);
-
-//! \todo  logik und Verwendung zone_fix vs zone_export unklar
-//! Was passiert, falls zonenummer in comment zu groess ist?
-	    if ((hr > 1) && (atoi(comment + hr - 1) != 0)) {
-		sprintf(zone, "%02d", atoi(comment + hr - 1));
-		strncpy(zone_fix, zone, 2);
-		zone_fix[2] = '\0';
-	    } else {
-		strncpy(zone_export, zone, 2);
-		zone_export[2] = '\0';
-	    }
-	}
-
-	if (strlen(hiscall) >= 2)
-	    OnLowerSearchPanel(32, zone_export);
-
-
-	for (ii = 0; ii < LEN(callpats); ii++) {
-
-	    hr = getlastpattern(callpats[ii]);	// call update ?
+    normalized_comment[0] = 0;
```

Comment:
These two lines are redundant. See above at line 737 and 738.

Reply:
Yes, forgot that I added them already.